trailblazer/apps/tower/api.py

```diff
@@ -207,3 +213,21 @@ def _get_job(self, task: TowerTask, analysis_id: int) -> dict:
             started_at=task.start,
             elapsed=int(task.duration / 60),
         )
+
+
+def _is_tower_api_client_requirements_meet(tower_api: TowerAPI) -> bool:
+    """Raises:
+    TowerRequirementsError when failing meeting Tower mandatory requirement"""
+    if not tower_api.tower_client.meets_requirements:
+        raise TowerRequirementsError("Could not initialize Tower API due to missing requirements")
+    return True
+
+
+def get_tower_api(config_file: str, case_id: str) -> Optional[TowerAPI]:
+    """Return Tower API. Currently only one tower ID is supported."""
+    workflow_id: int = ReadFile.get_content_from_file(
+        file_format=FileFormat.YAML, file_path=Path(config_file)
+    ).get(case_id)[-1]
+    tower_api = TowerAPI(workflow_id=str(workflow_id))
+    if _is_tower_api_client_requirements_meet(tower_api=tower_api):
```

Comment:
I would move the error rasing of function `_is_tower_api_client_requirements_met` here